src/app/query/bitcoin/balance/btc-balance.hooks.ts

```diff
@@ -6,10 +6,17 @@ import { createMoney } from '@shared/models/money.model';
 import { isUndefined } from '@shared/utils';
 
 import { sumNumbers } from '@app/common/math/helpers';
+import { useConfigRunesEnabled } from '@app/query/common/remote-config/remote-config.query';
+import { useCurrentNetwork } from '@app/store/networks/networks.selectors';
 
 import { useNativeSegwitUtxosByAddress } from '../address/utxos-by-address.hooks';
 
 export function useGetBitcoinBalanceByAddress(address: string) {
+  const runesEnabled = useConfigRunesEnabled();
+  const network = useCurrentNetwork();
+
+  const filterRunesUtxos = runesEnabled && network.chain.bitcoin.bitcoinNetwork === 'testnet';
```

Comment:
I don't think this condition is correct?

Reply:
yeah, fixed